packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -0,0 +1,271 @@
+    blockHash: transactionReceipt.result.blockHash,
+    transactionIndex: transactionReceipt.result.transactionIndex,
+    blockNumber: transactionReceipt.result.blockNumber,
+  };
+}
+
+async function checkRequestBody(fileName, request) {
+  //This method changes params for request which wont work with the values from ethereum api tests
+  if (
+    (request.method === 'eth_getBlockByHash' &&
+      request.params[0] === '0x7cb4dd3daba1f739d0c1ec7d998b4a2f6fd83019116455afa54ca4f49dfa0ad4') ||
+    (request.method === 'eth_sendRawTransaction' &&
+      request.params[0] ===
+        '0xf86709843b9aca018261a894aa000000000000000000000000000000000000000a825544820a95a0281582922adf6475f5b2241f0a4f886dafa947ecdc5913703b7840344a566b45a05f685fc099161126637a12308f278a8cd162788a6c6d5aee4d425cde261ba35d')
+  ) {
+    request.params[0] = currentBlockHash;
+  }
+  if (request.method === 'eth_getTransactionByBlockHashAndIndex') {
+    request.params[0] = legacyTransactionAndBlockHash.blockHash;
+    request.params[1] = legacyTransactionAndBlockHash.transactionIndex;
+  }
+  if (request.method === 'eth_getTransactionByBlockNumberAndIndex') {
+    request.params[0] = legacyTransactionAndBlockHash.blockNumber;
+    request.params[1] = legacyTransactionAndBlockHash.transactionIndex;
+  }
+  if (request.method === 'eth_sendRawTransaction') {
+    const transactionCount = await getTransactionCount();
+    legacyTransaction.nonce = parseInt(transactionCount, 16);
+    const transactionHash = await signTransaction(
+      legacyTransaction,
+      '0x6e9d61a325be3f6675cf8b7676c70e4a004d2308e3e182370a41f5653d52c6bd',
+    );
+    request.params[0] = transactionHash;
+  }
+  if (request.method === 'eth_getBalance') {
+    request.params[0] = '0x5C41A21F14cFe9808cBEc1d91b55Ba75ed327Eb6';
+    request.params[1] = currentBlockHash;
+  }
+  if (request.method === 'eth_getTransactionByHash' || request.method === 'eth_getTransactionReceipt') {
+    request = formatTransactionByHashAndReceiptRequests(fileName, request);
+  }
+  return request;
+}
+
+function checkResponseFormat(fileName, actualReponse, expectedResponse) {
+  const actualResponseKeys = extractKeys(actualReponse);
+  const expectedResponseKeys = extractKeys(expectedResponse);
+  const missingKeys = expectedResponseKeys.filter((key) => !actualResponseKeys.includes(key));
+  if ((fileName === 'get-dynamic-fee.io' || fileName === 'get-access-list.io') && missingKeys[0] === 'result.v') {
+    return [];
+  }
+
+  return missingKeys;
+}
+
+function extractKeys(obj, prefix = '') {
+  let keys = [];
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      const newKey = prefix ? `${prefix}.${key}` : key;
+      keys.push(newKey);
+
+      if (typeof obj[key] === 'object' && !Array.isArray(obj[key])) {
+        keys = keys.concat(extractKeys(obj[key], newKey));
+      }
+    }
+  }
+
+  return keys;
+}
+
+function formatTransactionByHashAndReceiptRequests(fileName, request) {
+  switch (fileName) {
```

Comment:
WHat are these file name and where are they from?
Maybe add comment but also if possible make them constants

Reply:
They are taken on the go, so cannot make the constants
This is the fileName of each file we process from the execution api tests

Reply:
I'm not clear why the case values of the fileName can't be constants.
For instance `get-dynamic-fee.io` and `get-access-list.io` are used twice in this file.
It'd make it more manageable to have a section at the top of all the fileNames.

Reply:
Sorry, now I got what you meant :) Hope its okay now